bin/code_tnhaplotyper2.sh

```
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+fastaIndex="$1"
+gatkResource="$2"
+bam="$3"
+bam_bai="$4"
```

Shellcheck (0.10.0)

[warning] bam_bai appears unused. Verify use (or export if used externally).

(SC2034)